frontend/src/lib-components/atoms/RoundIcon.tsx

```diff
@@ -59,32 +59,35 @@ const IconContainer = styled.div<IconContainerProps>`
     font-size: 8px;
     ${diameter(16)}
   }
+
   &.s {
     font-size: 12px;
     ${diameter(20)}
   }
+
   &.m {
     font-size: 16px;
-    ${diameter(24)}
 
+    ${diameter(24)}
     span.text {
       font-family: Montserrat, sans-serif;
       font-weight: ${fontWeights.bold};
     }
   }
+
   &.l {
     font-size: 18px;
     ${diameter(34)}
   }
+
   &.xl {
     font-size: 44px;
-    ${diameter(64)}
-
-    @media (max-width: ${tabletMin}) {
+    ${diameter(64)} @media(max-width: ${tabletMin}) {
```

Comment:
Formatting seems to be a little bit off here.

Reply:
This is some weird IntelliJ IDEA bug I think. It wants to format CSS strings and doesn't understand rows that don't end in a `;`

Reply:
Fixed